nodejs/week1/src/backend/routes/meals.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const dataAccess = require("../dataAccess.js");
+const router = express.Router();
+const meals = "/data/meals.json";
+const reviews = "/data/reviews.json";
```

Comment:
Remove. Same explanation than the previous file. Also export only router function. Import json using require directly.